_posts/2016-10-05-bio-formats-5.2.3.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title: Release of Bio-Formats 5.2.3
+intro-blurb: The OME team is pleased to announce the release of Bio-Formats 5.2.3
+---
+
+Today we are releasing Bio-Formats 5.2.3, a bug-fix release. 
+
+Improvements include fixes for three formats:
+
+*  CZI
+    -  fixed imageCount for RGB images
+*  ICS writing
+    -  fixed ordering of image dimensions
+*  DeltaVision
+    -  fixed reading of large time dimensions
+
+Other improvements:
+
+*  Command-line tools
+    -  bftools.zip now includes the version history as NEWS.rst (thanks to Gerhard Burger)
+*  Code clean-up
+    -  switched to String.indexOf(int) in GPL-licensed reader code so that a simpler library method can be used
+    -  strings now extended with characters where possible
+    -  completed deprecation of DataTools.sanitizeDouble()
+    -  deprecated unused OSGi and ome-tools bundle build targets
+*  Documentation
+    -  documented versioning policy
+    -  clarified supported versions for Micro-Manager and Olympus ScanR files
+*  OME-XML changes
+    -  bumped schema version number to 2 (schema namespace left unchanged)
+    -  added acquisition modes BrightField, SweptFieldConfocal and SPIM
+    -  added parsing for Laser Scan Confocal and Swept Field Confocal
+
+The software is available to download now at 
+[http://downloads.openmicroscopy.org/bio-formats/5.5.1](http://downloads.openmicroscopy.org/bio-formats/5.5.1).
```

Comment:
Should be [archived downloads] cf https://github.com/openmicroscopy/www.openmicroscopy.org/pull/53/files#diff-639b1eb8b9357fce496f08267c893ef8R36